components/CSVTable.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react'
+import Table from '@mui/material/Table'
+import TableBody from '@mui/material/TableBody'
+import TableCell from '@mui/material/TableCell'
+import TableContainer from '@mui/material/TableContainer'
+import TableHead from '@mui/material/TableHead'
+import TableRow from '@mui/material/TableRow'
+import Paper from '@mui/material/Paper'
+
+function createData (
+  email: string,
+  subject: string,
+  name: string,
+  company: string
+) {
+  return { email, subject, name, company }
+}
+
+const rows = [
+  createData('example@facebook.com', 'Sponsoring HackBeanpot', 'Danaerys', 'Facebook'),
+  createData('example@google.com', 'Sponsoring HackBeanpot', 'Jaime', 'Google'),
+  createData('example@appfolio.com', 'Sponsoring HackBeanpot', 'Tyrion', 'Appfolio')
+]
+
+export default function CSVTable () {
+  return (
+    <TableContainer component={Paper}>
+      <Table aria-label="simple table">
+        <TableHead>
+          <TableRow>
+          <TableCell align="left">email</TableCell>
+            <TableCell align="left">subject</TableCell>
+            <TableCell align="left">name</TableCell>
+            <TableCell align="left">company</TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {rows.map((row) => (
```

Comment:
We should do the same thing here as the headers and pass in rows into this component. So instead of creating the rows here you could create them in the help page and pass them into this component as props.